openstack/objectstorage/v1/objects/requests.go

```diff
@@ -464,7 +474,7 @@ func CreateTempURL(c *gophercloud.ServiceClient, containerName, objectName strin
 		opts.Split = "/v1/"
 	}
 	duration := time.Duration(opts.TTL) * time.Second
-	expiry := time.Now().Add(duration).Unix()
+	expiry := timeNow().Add(duration).Unix()
```

Comment:
I wonder how did it work before without `UTC()`

Reply:
Good question, I didn't think about htat. I was reading docs, and it says it should be consistent in any way :arrow_down:.

> `Unix()` The result does not depend on the location associated with t. Unix-like operating systems
https://golang.org/pkg/time/#Time.Unix

So I can actually revert back to `Now()` if it looks better for you.

Reply:
https://play.golang.org/p/so0mPFXWY7m

Reply:
Not sure what should be used. @jtopjian please advice.

Reply:
Can we resolve as discussed below? 
It was commited here https://github.com/gophercloud/gophercloud/commit/9a500de9eaad35ab3e640fe323ccfe47bf88532d